homeassistant/components/growatt_server/config_flow.py

```diff
@@ -0,0 +1,78 @@
+"""Config flow for growatt server integration."""
+import voluptuous as vol
+
+import growattServer
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_NAME, CONF_USERNAME, CONF_PASSWORD
+from .const import DOMAIN, CONF_PLANT_ID, DEFAULT_NAME
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class GrowattServerConfigFlow(config_entries.ConfigFlow):
```

Comment:
Pass the domain to the domain keyword argument and remove the decorator.

https://developers.home-assistant.io/docs/en/config_entries_config_flow_handler.html#defining-your-config-flow